packages/utils/src/OptionalIntUtil.ts

```diff
@@ -0,0 +1,37 @@
+import { OptionalInt } from '@syndesis/models';
+
+export class OptionalIntUtil {
```

Comment:
I'd dump the class since it contains only static methods, and export the functions directly.

Reply:
I'll remove the class and export the 3 functions.